### PR TITLE
added attr parsing support

### PR DESCRIPTION
**Pull request by maZahaca**
* added attr parsing support
* added _id row support

#49, #50 

For now in simple rule we can set:
* attr: 'attr-name' - if specified will be parsed from attribute, otherwise from textContent
* id: true | Function, will be set with name _id: {parsing result or result of function call}

### CODE REVIEW

**Comment by maZahaca** [comment]
@jifeon for review

File: README.md [hunk 1]
```diff
@@ -236,6 +236,8 @@ The purpose of this rule - retrieving simple textual node value(s).
 
 * *name* - name of the node which is presented in the result dataSet.
 * *scope* - css selector of the node.
+* *id* [optional] - true|Function. Will mark a row as identifier, the result will be set to node with `_id` name. If function specified, parser will call it for each row.
```
Comment (jifeon) [inline README.md]: please add real life explanation of this feature

File: lib/Environment.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ var vow = require('vow'),
     debug = require('debug')('Environment');
 
 function Environment(options) {
-    debug('Initializing..');
+    debug('Initializing...');
```
Comment (jifeon) [inline lib/Environment.js]: :+1:

File: lib/Parser.js [hunk 3]
```diff
@@ -374,8 +376,20 @@ Parser.prototype = {
...
-                    results[rule.name] = result;
-                });
+                    var name;
+                    switch (typeof rule.id) {
+                        case 'boolean':
+                            name = '_id';
+                            break;
+                        case 'function':
+                            name = '_id';
+                            result = rule.id.call(this, rule, result);
```
Comment (jifeon) [inline lib/Parser.js]: there is nothing about these arguments in docs

File: lib/Parser.js [hunk 4]
```diff
@@ -390,12 +404,18 @@ Parser.prototype = {
...
         var selector = this._getSelector();
+        var attr = rule.attr || null;
         rule.type = rule.type || 'string';
         debug('._parseSimpleRule() has called for selector %s', selector);
-        return this._env.evaluateJs(selector, /* @covignore */ function (selector) {
+        return this._env.evaluateJs(selector, attr, /* @covignore */ function (selector, attr) {
             var nodes = Sizzle(selector);
             return nodes.map(function (node) {
-                return node.textContent;
+                if (attr !== null) {
```
Comment (jifeon) [inline lib/Parser.js]: just note: I would use ternary operator here

File: package.json [hunk 5]
```diff
@@ -10,8 +10,14 @@
...
     "crawler",
+    "crawling",
+    "spider",
+    "scraper",
+    "scraping",
     "site",
-    "page"
+    "page",
+    "dom",
+    "javascript"
```
Comment (jifeon) [inline package.json]: free, without SMS

**Comment by jifeon** [comment]
cool stuff!

**Pull request merged**